### DISCUSSION TITLE
Fluxstore Listing - click to fullscreen listing images

### DISCUSSION THREAD

**Discussion by merorose** [inspireui/support#5779] [New Feature/Ideas Requests] [answered]
in fluxstore woo commerce there is an option on product page when click product image it open the image in fullscreen mood with ability to zoom-in and out
this option is not applied for fluxstore listing
we need same option , where user can click listing images to view in fullscreen mode with ability to zoom

Uploading RPReplay_Final1702032931.mov…

**Comment by hoangphuc1103** [comment, marked as answer]
Thank you for your contribution; this is now available and supports in the Fluxstore Listing app 🎉
